packages/client/shared/gqlIds/PollId.ts

```diff
@@ -0,0 +1,9 @@
+const PollId = {
+  create: (pollId: number) => `poll:${pollId}`,
```

Comment:
nit: the other files in the folder have a `join` method. I know where' not actually joining anything here though. not sure which is better :shrug:

Reply:
Join it is!